### DISCUSSION TITLE
Background color option

### DISCUSSION THREAD

**Discussion by Ctrl-sza18** [fast4x/RiMusic#3861] [Ideas]
Yo I wanted to ask if it was possible to add color palettes for the album or single cover and make it a option to change it for fluid background or gradient eg. 
![Screenshot_20241003_171558_com.toasterofbread.spmp.jpg](https://github.com/user-attachments/assets/6b7dd250-37e8-4ac3-81e7-28b7626c4010)

**Comment by fast4x** [comment]
gradient and blurred are just implemented, you can found it in player appearance settings.
